neurots/generate/orientations.py

```diff
@@ -360,3 +438,216 @@ def compute_interval_n_tree(soma, n_trees, rng=np.random):
         interval_n_trees = np.array([n_trees])
 
     return phi_intervals, interval_n_trees
+
+
+def spherical_angles_to_pia_orientations(phis, thetas):
+    """Compute orientation from spherical angles where thetas are wrt to pia at `[0, 1, 0]`.
+
+    Args:
+        phis (numpy.ndarray): Polar angles.
+        thetas (numpy.ndarray): Azimuthal angles.
+
+    Returns:
+        numpy.ndarray: The orientation vectors where each row corresponds to a phi-theta pair.
+    """
+    return np.column_stack(
+        (np.cos(phis) * np.sin(thetas), np.cos(thetas), np.sin(phis) * np.sin(thetas))
+    )
```

Comment:
I think this function should go in `morphmath.rotation`. But maybe with a warning in the docstring to clearly explain that this function is only relevant with angles wrt to pia direction.

Reply:
well, there is spherical_angles_orientation in this module, that's why it is here